test/functionalTest/cases/0000_ngsi10_convenience/subscriptions_onchange.test

```diff
@@ -29,7 +29,6 @@ brokerStart CB
 accumulatorStart
```

Comment:
I'd suggest to get a copy of the original subscriptions_onchange.test file (e.g. subscriptions_onchange_nocache.test) in order to ensure that the -noCache functionaly still working in the old way (subscriptions_onchange_nocache.test would need an adjustment in the brokerStart line to use the -noCache option, of course).